### PR TITLE
Test app session now replay Toast notifications

### PR DESCRIPTION
**Pull request by bstevensmetova**

### CODE REVIEW

File: Example/NativeBase-KitchenSink/src/screens/toast/basic-toast.js [hunk 1]
```diff
@@ -41,7 +42,10 @@ class BasicToast extends Component {
             onPress={() =>
               Toast.show({
                 text: "Wrong password!",
-                buttonText: "Okay"
+                buttonText: "Okay",
+                onClose: () => {
```
Comment (ohernandezIBM) [inline Example/NativeBase-KitchenSink/src/screens/toast/basic-toast.js:46]: Is there a way to capture automatically? So customer does not need to add one line in onClose?

File: Example/NativeBase-KitchenSink/src/screens/toast/toast-button.js [hunk 2]
```diff
@@ -13,45 +13,43 @@ import {
...
+
+      <Content padder>
+        <Button
+          onPress={() => {
+            Toast.show({
+              text: "Wrong password!",
+              buttonText: "Okay",
+              buttonTextStyle: { color: "#008000" },
+              buttonStyle: { backgroundColor: "#5cb85c" },
+              onClose: () => {
```
Comment (ohernandezIBM) [inline Example/NativeBase-KitchenSink/src/screens/toast/toast-button.js:42]: Is there a way to capture automatically? So customer does not need to add one line in onClose?

File: Example/NativeBase-KitchenSink/src/screens/toast/toast-duration.js [hunk 3]
```diff
@@ -42,7 +43,10 @@ class ToastDuration extends Component {
               Toast.show({
                 text: "Wrong password!",
                 buttonText: "Okay",
-                duration: 3000
+                duration: 3000,
+                onClose: () => {
```
Comment (ohernandezIBM) [inline Example/NativeBase-KitchenSink/src/screens/toast/toast-duration.js:47]: Is there a way to capture automatically? So customer does not need to add one line in onClose?

File: Example/NativeBase-KitchenSink/src/screens/toast/toast-position.js [hunk 4]
```diff
@@ -42,7 +43,10 @@ class ToastPosition extends Component {
               Toast.show({
                 text: "Wrong password!",
                 buttonText: "Okay",
-                position: "top"
+                position: "top",
+                onClose: () => {
+                  TLTRN.logScreenLayout("Wrong password!");
```
Comment (ohernandezIBM) [inline Example/NativeBase-KitchenSink/src/screens/toast/toast-position.js:48]: Is there a way to capture automatically? So customer does not need to add one line in onClose?

File: Example/NativeBase-KitchenSink/src/screens/toast/toast-position.js [hunk 5]
```diff
@@ -53,7 +57,10 @@ class ToastPosition extends Component {
               Toast.show({
                 text: "Wrong password!",
                 buttonText: "Okay",
-                position: "bottom"
+                position: "bottom",
+                onClose: () => {
+                  TLTRN.logScreenLayout("Wrong password!");
```
Comment (ohernandezIBM) [inline Example/NativeBase-KitchenSink/src/screens/toast/toast-position.js:62]: Is there a way to capture automatically? So customer does not need to add one line in onClose?

File: Example/NativeBase-KitchenSink/src/screens/toast/toast-type.js [hunk 6]
```diff
@@ -41,7 +42,10 @@ class ToastType extends Component {
             onPress={() =>
               Toast.show({
                 text: "Wrong password!",
-                buttonText: "Okay"
+                buttonText: "Okay",
+                onClose: () => {
+                  TLTRN.logScreenLayout("Wrong password!");
```
Comment (ohernandezIBM) [inline Example/NativeBase-KitchenSink/src/screens/toast/toast-type.js:47]: Is there a way to capture automatically? So customer does not need to add one line in onClose?

**Review by ohernandezIBM — APPROVED** [review]
Is there a way to capture automatically? So customer does not need to add one line in onClose?

File: Example/NativeBase-KitchenSink/src/screens/toast/basic-toast.js (hunk 1 above)
Comment (bstevensmetova) [reply Example/NativeBase-KitchenSink/src/screens/toast/basic-toast.js:46]: There's no listener to be able to capture something like that, plus I don't think we can assume what toast notification system the customer is using. The sample app is using the native-base package to show toast notifications while other apps might be using something like [react-native-toast-notifications](https://www.npmjs.com/package/react-native-toast-notifications) or even a custom implementation.

**Pull request merged**